### PR TITLE
Add code example

### PR DESCRIPTION
**Pull request by francesco-strazzullo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# ButterCMS Alexa Skill Example
```
Comment (jakelumetta) [inline README.md]: Can you add some high level getting started steps here for other folks that want to try using this skill?

Comment (francesco-strazzullo) [reply README.md]: Ok, I will add some info on the README!

Comment (jakelumetta) [reply README.md]: Thanks - please let me know once it's ready and I'll take a final look.

Comment (francesco-strazzullo) [reply README.md]: @jakelumetta done!

**Pull request merged**